### PR TITLE
Replace UseImportExportParams with FlatParameters

### PR DESCRIPTION
**Pull request by trikimahdi**

### CODE REVIEW

File: src/components/dialogs/export-dialog.js [hunk 1]
```diff
@@ -69,13 +68,26 @@ const ExportDialog = ({
 
     const formatWithParameter = formatsWithParameters?.[selectedFormat];
     const metasAsArray = formatWithParameter?.parameters || [];
-    const [currentParameters, paramsComponent] = useImportExportParams(
-        metasAsArray,
-        null,
-        null,
-        'standard'
+    const [currentValues, setCurrentValues] = useState({});
+    const currentParameters = {};
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/export-dialog.js:72]: remove this unused empty object

File: src/components/dialogs/export-dialog.js [hunk 2]
```diff
@@ -69,11 +68,24 @@ const ExportDialog = ({
...
+        if (!isEdit) {
+            setCurrentParameters((prevCurrentParameters) => {
+                return {
+                    ...prevCurrentParameters,
+                    ...{ [paramName]: value },
+                };
+            });
+        }
+    }, []);
+    const paramsComponent = (
```
Comment (sBouzols) [inline src/components/dialogs/export-dialog.js:82]: Move it in the JSX ?

File: src/components/dialogs/export-dialog.js [hunk 3]
```diff
@@ -69,11 +68,24 @@ const ExportDialog = ({
 
     const formatWithParameter = formatsWithParameters?.[selectedFormat];
     const metasAsArray = formatWithParameter?.parameters || [];
-    const [currentParameters, paramsComponent] = useImportExportParams(
-        metasAsArray,
-        null,
-        null,
-        'standard'
+    const [currentParameters, setCurrentParameters] = useState({});
```
Comment (sBouzols) [inline src/components/dialogs/export-dialog.js:71]: call `setCurrentParameters({})` in `handleClose`

**Review by sBouzols — APPROVED** [review]
Code review ok

File: src/components/dialogs/parameters/load-flow-parameters.js [hunk 4]
```diff
@@ -148,6 +146,90 @@ const DoubleEditor = ({
...
+
+function makeDeltaMap(defaultMap, changingMap) {
+    if (!changingMap) {
+        return null;
+    }
+
+    const delta = {};
+
+    Object.entries(defaultMap).forEach(([k, v]) => {
+        const m = changingMap[k];
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/parameters/load-flow-parameters.js:170]: rename to more significant name

File: src/components/dialogs/export-dialog.js (hunk 1 above)
Comment (trikimahdi) [reply src/components/dialogs/export-dialog.js]: done

**Review by Walid-Sahnoun — APPROVED** [review]
review ok

**Pull request merged**